### DISCUSSION TITLE
Strip (or modify) SNI from request

### DISCUSSION THREAD

**Discussion by biennvops** [Floorp-Projects/Floorp#454] [Feature Request]
My country has DPI-based website blocking which is annoying because it blocks even useful websites, and right now there isn't any browsers that support removing SNI (except for legacy browsers which aren't ideal for daily driver).
I'm using macOS, and setting up a proxy using apps like PowerTunnel will prevent VPN from working properly (Private Relay for example will turn off when I set a system-wide proxy).

And yes, I know I have to use a VPN to hide my actual traffic data, I just want to access websites without having to pay for one (I bought icloud+ for private relay which is actually pretty cheap because of this).

**Comment by gunir** [comment]
This feature can be good too, if it's possible we should try to implement it as a include/exclude settings

**Comment by Ryosuke-Asano** [comment]
This is strictly Floorp policy as it constitutes a deviation from web standards.
